www/js/app.js

```diff
@@ -416,6 +419,16 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         // Do the necessary to enable or disable the Service Worker
         setContentInjectionMode(this.value);
     });
+    document.getElementById('useCanvasElementsCheck').addEventListener('change', function () {
+        if (this.checked) {
+            settingsStore.removeItem('useCanvasElements');
+            params.useCanvasElements = uiUtil.determineCanvasElementsWorkaround();
```

Comment:
The responsibilities of this function `uiUtil.determineCanvasElementsWorkaround` is not clear to me.
It seems to me that the function sets `params.useCanvasElements`, but also returns its value, with is set again here.

Reply:
Yes you are right. I changed it recently so that the function sets params.useCanvasElements so that we didn't end up running it twice on every app load. And I forgot to remove the assignment here. This should just run the function, not set it to the result.